float64 v_bat
float64 i_bat
float64 v_pv
float64 p_pv
